src/app/[locale]/(main)/page.tsx

```diff
@@ -15,16 +60,45 @@ export default function Home() {
     <div className="min-h-screen">
       <div className="relative flex h-[70vh] w-full flex-col items-center justify-center overflow-hidden">
         <div className="pointer-events-none absolute inset-0 z-10 h-full w-full bg-background [mask-image:linear-gradient(to_top_right,white_40%,transparent_50%)]" />
-        <Boxes row={60} className="to-[-30%]" />
+        <Boxes row={60} className="-top-[30%]" />
         <p className="relative z-20 bg-gradient-to-b from-slate-400 to-slate-800 bg-clip-text font-heading text-4xl font-black text-transparent sm:text-7xl">
           {t("title")}
         </p>
         <p className="relative z-20 bg-gradient-to-b from-slate-300 to-slate-700 bg-clip-text text-sm font-black tracking-wider  text-transparent sm:text-base ">
           Bring joy and fun to your classroom
         </p>
+        <LoginCard />
       </div>
 
-      <div className="space-y-4 text-center">
+      {/* Recent and stats */}
+      <div className="bg-[#E5E5E5] p-10">
+        <div className="p-2">
+          <div className="mb-2 underline">Recent</div>
+          <div className="flex justify-between space-x-4">
+            {cardInfos.map((card, i) => (
+              <CardInfo
+                className="h-[14vh] max-h-[20vh] w-[21.9vw] max-w-[40vw]"
```

Comment:
This not working on many screen size.  Maybe set a ratio or use strict values to keep shape 
Examples
On large screen:
![image](https://github.com/blackmouse572/quizler-fe/assets/77144199/21927233-09db-4879-8324-f0c1676bfa25)
on less height screen
![image](https://github.com/blackmouse572/quizler-fe/assets/77144199/7a467bf8-524f-489c-bd3b-770aa904c5e0)